src/main/webapp/app/modules/home/ActivityElement.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Row, Col, Card, CardText, CardBody, CardTitle, CardGroup, Badge } from 'reactstrap';
+import { Translate, TextFormat } from 'react-jhipster';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { APP_DATE_FORMAT } from 'app/config/constants';
+
+const ActivityElement = props => (
+  <Row style={{ marginBottom: '15px' }}>
+    <Col>
+      <CardGroup>
+        <Card style={{ marginBottom: '0px' }}>
+          <CardBody>
+            <CardTitle style={{ color: '#66c4ff' }}>
+              {props.activity.organization.name} <FontAwesomeIcon icon="angle-right" />
+            </CardTitle>
+            <CardTitle>
+              <FontAwesomeIcon icon="map-marker-alt" /> {props.activity.organization.locationName}
+            </CardTitle>
+          </CardBody>
+        </Card>
+        <Card style={{ backgroundColor: '#f4f4f4' }}>
+          <CardBody>
+            {props.activity.conflicts.map(conflict => (
+              <CardTitle style={{ color: '#66c4ff' }}>
+                {conflict.ownerName} {conflict.acceptedThisChanges.length > 0 && <b>+{conflict.acceptedThisChanges.length}</b>}
+                {conflict.acceptedThisChanges.length > 1 && (
+                  <Translate
+                    contentKey="serviceNetApp.activity.unresolved.conflictsPlusMany"
+                    interpolate={{ fieldName: conflict.fieldName }}
+                  >
+                    partners have a conflicting {conflict.fieldName}
+                  </Translate>
+                )}
+                {conflict.acceptedThisChanges.length === 1 && (
```

Comment:
You could put those components as a variables above, as it would be more readable:

`let conflictsPlusMany =  <Translate
                    contentKey="serviceNetApp.activity.unresolved.conflictsPlusMany"
                    interpolate={{ fieldName: conflict.fieldName }}
                  >
                    partners have a conflicting {conflict.fieldName}
                  </Translate>`
and then something like
`{conflict.acceptedThisChanges.length > 1 && conflictsPlusMany`

Reply:
I cannot because it's a list's variable.